app/views/vm_common/console_vmrc.html.haml

```diff
@@ -367,36 +367,42 @@
         // Start up VMRC
         vmrc = document.getElementById("vmrc");
 
-        vmNameTooltip = $("#vmNameTooltip");
-        var vmTitle = $("#vmTitle");
+        if (typeof vmrc.isReadyToStart === 'undefined') {
+          window.location.href = "vmrc://clone:#{ticket}@#{host}:443/?moid=#{vmid}";
```

Comment:
If you want use vmrc:// as the scheme, then use URI::Generic and pass the scheme manually...

```ruby
URI::Generic.build(:scheme => "vmrc", :userinfo => "clone:#{ticket}", :host => "::1", :port => 443, :path => "/", :query => "moid=#{vmid}")